packages/components/navbar/src/Navbar.tsx

```diff
@@ -48,12 +51,14 @@ function _Navbar(props: ExpandProps<NavbarProps>, ref: React.Ref<HTMLElement>) {
     topRightItems,
     contentMaxWidth = '100%',
     testId = 'cf-ui-navbar',
+    className,
+    variant = 'wide',
```

Comment:
Although `wide` should be our default, we will have a much easier time migrating to the new Layout if we go with `fullscreen` for now. I'm a bit torn, but since it's an alpha I'm thinking we can always switch it to `wide` later on. What do you think?

Reply:
I would rather do the necessary changes now than executing a code-mod later and would go directly with the wide variant as default. 
At least this way the fullscreen is a conscious decision from the beginning and we can challenge if its even needed, when it appears to us.
While if we change it later via a code mod we will not think twice over the usage of the fullscreen variant. 

But maybe @damann can weight in if fullscreen should be the exception or the norm?

Reply:
We need `fullscreen` everywhere we don't have the page cut. So everywhere but 4 screens.